### ISSUE TITLE
About calculating 'thres' in unsupervised loss.

### ISSUE THREAD

**Issue by DISAPPEARED13** [Haochen-Wang409/U2PL#103]
Hi there.
I found that the code in computing unsupervised loss:
`thresh = np.percentile(entropy[target != 255].detach().cpu().numpy().flatten(), percent)`

And, sometimes, my network will generate all-zero output, should I just add a try-except to skip calculating unsupervised loss? Or it's just because some bugs in code?

Thanks a lot.

**Comment by Haochen-Wang409** [comment]
We define ```sup_only_epoch``` in our experiments, which means during these epochs, only supervised loss is backpropagated. Therefore, unsupervised loss and contrastive loss are expected to be zero at the beginning.
